main.js

```diff
@@ -0,0 +1,62 @@
+const showField = document.querySelector('.books-list');
+const submit = document.getElementById('submit');
+
+const bookList = [];
+
+function Voks(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function saveLocal() {
+  localStorage.setItem('data', JSON.stringify(bookList));
+}
+
+function addBook() {
+  const newBook = new Voks();
+  newBook.title = document.getElementById('title').value;
+  newBook.author = document.getElementById('author').value;
+  bookList.push(newBook);
+}
+
+function pushContent() {
+  document.getElementById('books-list').innerHTML = '';
+  document.getElementById('title').value = '';
+  document.getElementById('author').value = '';
+  bookList.forEach((e) => {
+    const contain = document.createElement('div');
+    contain.className = 'contain';
+    const title = document.createElement('div');
+    title.className = 'title';
+    const author = document.createElement('div');
+    author.className = 'author';
+    title.innerHTML = e.title;
+    author.innerHTML = e.author;
+    const remove = document.createElement('button');
+    remove.className = 'remove';
+    remove.textContent = 'Remove';
```

Comment:
- [ ] The localstorage looks good, but the UI doesn't showing.

![Screenshot from 2021-09-21 21-07-22](https://user-images.githubusercontent.com/22434660/134203255-d042b014-b6dc-4932-bf70-7f6300f3337a.png)

Reply:
can you provide more details about that point, please?

Reply:
Hi @ahmedadel56,

When I try to refresh the page, the added book will remain in localstorage, but it vanishes from the UI. So, make sure you would retrieve the data from localstorage and show it in the screen.

Sorry for not explaining before. Hope you found it well explained.

Thanks